### PR TITLE
Module 1 small fixes

### PR DESCRIPTION
**Pull request by mghadam**

### CODE REVIEW

File: lessons/01_phugoid/01_03_PhugoidFullModel.ipynb [hunk 1]
```diff
@@ -729,7 +729,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "* Feng, N. B. et al. _\"On the aerodynamics of paper airplanes\"_, AIAA paper 2009-3958, 27th AIAA Applied Aerodynamics Conference, San Antonio, TX. [PDF](http://www.ntu.edu.sg/home/schluter/pdf/aiaa-2009-3958.pdf)\n",
+    "* Feng, N. B. et al. _\"On the aerodynamics of paper airplanes\"_, AIAA paper 2009-3958, 27th AIAA Applied Aerodynamics Conference, San Antonio, TX. [PDF](http://arc.aiaa.org/doi/abs/10.2514/6.2009-3958)\n",
```
Comment (gforsyth) [inline lessons/01_phugoid/01_03_PhugoidFullModel.ipynb]: Could you replace this with this URL instead: https://www.researchgate.net/publication/261948508_On_the_Aerodynamics_of_Paper_Airplanes

That's a free copy put up by the author.  Thanks for spotting that the link was bad, but we don't want to link to a paywall.

**Comment by gforsyth** [comment]
Thanks @mghadam!  If you could just update the link I posted in the line comments I'm happy to merge this in.

**Comment by labarba** [comment]
Actually, I'm not a big fan of ResearchGate and would rather not send traffic their way.

By linking to the paywalled article, at least is the official publication and everyone on the 'net knows how to get a free copy if they want to.

**Comment by gforsyth** [comment]
Fair enough.  Random Google Scholar PDF link ok? http://www.enu.kz/repository/2009/AIAA-2009-3958.pdf

**Comment by mghadam** [comment]
I first wanted to put the enu.kz link but thought it has copyright issues.

I think students with no university access will search for the pdf file and find a direct link link themselves.

**Comment by gforsyth** [comment]
Or we can link to the google scholar entry?

**Comment by mghadam** [comment]
I prefer the current AIAA link as modifying the commit history and updating the link is not straightforward 😁

**Comment by gforsyth** [comment]
Fine by me

**Comment by gforsyth** [comment]
Thanks, @mghadam !

**Comment by mghadam** [comment]
You're welcome.

@gforsyth I think there's an issue with the module 1 code assignment (Rocket flight). How can we discuss it? Shall I open an issue and put my code there to discuss it?

**Comment by gforsyth** [comment]
That would be fine.  If possible, try to only post the relevant snippets.

**Pull request merged**